schema/sqitch.plan

```diff
@@ -214,3 +214,7 @@ receiving/presence-absence/indexes/processing-log [receiving/presence-absence] 2
 receiving/redcap-det/indexes/processing-log [receiving/redcap-det] 2020-11-16T20:54:38Z Thomas Sibley <tsibley@fredhutch.org> # Index receiving.redcap_det.processing_log column
 receiving/sequence-read-set/indexes/processing-log [receiving/sequence-read-set] 2020-11-16T20:54:41Z Thomas Sibley <tsibley@fredhutch.org> # Index receiving.sequence_read_set.processing_log column
 @2020-11-17 2020-11-17T18:24:24Z Thomas Sibley <tsibley@fredhutch.org> # Schema as of 17 Nov 2020
+
+functions/barcode_slices 2020-11-17T18:31:45Z Chris Craft <jccraft@uw.edu> # Add function to create slices out of input text
+warehouse/identifier/slices 2020-11-17T18:32:16Z Chris Craft <jccraft@uw.edu> # Add slices column to warehouse.identifier table
+warehouse/identifier/triggers/barcode-distance-check [warehouse/identifier/triggers/barcode-distance-check@2020-11-17] 2020-11-17T18:36:26Z Chris Craft <jccraft@uw.edu> # Add slices check to barcode-distance-check
```

Comment:
As a dot the i's and cross the t's nit: this last Sqitch change has a dependency on the one before it, `warehouse/identifier/slices`, but that's not declared in the plan. You can just add it manually to the dep list.

Reply:
Gotcha. I've added that in the fixup. I also added a required to warehouse/identifier/slices.sql ... it needs the function.

Reply:
In the new index implementation both the index and the barcode-distance-check depend on the function. I've added that in for those 2.

Reply:
Ah, I see the informational `-- requires: …` comments you added. Those are good, but are informational only (e.g. for humans only). Sorry, I should have been clearer: the place that matters is in _schema/sqitch.plan_ within the square brackets, i.e. the line I made my original review comment on.

Reply:
Ah, sorry about that. I've added the dependencies in the sqitch.plan too.